### ISSUE TITLE
[BUG] no information of selected shipping option by buyer

### ISSUE THREAD

**Issue by tlesiuk** [etsy/open-api#124]
In order of investigation how Etsy API works, we found that it is not possible to get information which type of shipping delivery buyer selects.
I mean that https://www.etsy.com/openapi#operation/getListing returns only shipping_template_id, which refers to shipping profile.
The shipping profile also provides information about delivery to UK, everywhere and shipping upgrades with few options.
But using “shipping_template_id” we can read information of every listing using Etsy API and we can’t know what type of shipping delivery buyer selects.

Could you please add or update this functionality to Etsy API v.3 to get information which type of shipping delivery buyer select. It will be very helpful for us.

**Comment by amandalik** [comment]
@etsyachristensen is there an estimated date by which this will be resolved?

**Comment by etsyachristensen** [comment]
There's no date set yet. We put a lot into just getting the initial public launch out. Now that this is done we are regrouping on these issues to determine what items are in which priority. But I can tell you that this is very high on the list.

**Comment by amandalik** [comment]
hey @etsyachristensen, just reaching out again to see if this has been looked into. This is business crucial for us and we would like to get this resolved as soon as possible!

**Comment by amandalik** [comment]
@etsyachristensen following up again, please let me know if this has been prioritized.

**Comment by etsyachristensen** [comment]
@amandalik Yes. This is actually in process now. I'm hoping to have it completed and in production this week.

**Comment by tanishqandmac** [comment]
@etsyachristensen any update on this?

**Comment by amandalik** [comment]
@etsyachristensen any progress on this?

**Comment by etsyachristensen** [comment]
Yes @amandalik . The transaction data that is in the receipt now includes shipping_method, expected_ship_date and actual_ship_date fields. We are working on adding a bit more data, but it's a more complicated process.

**Comment by tlesiuk** [comment]
Thank you, @etsyachristensen, for update and we've already integrated transaction data from receipt list in our app. But, still, we can find relationship between type of delivery (Standart shipping, Ecenomy etc.) and carrier name. Have you plan to add this in API?

**Comment by tlesiuk** [comment]
@etsyachristensen But if seller doesn't set shipping profile in the listing, in that case we can't read what carrier is related to shipping method. It seems incomprehensible for me.

**Comment by etsyachristensen** [comment]
@tlesiuk There has to be a shipping profile for each listing unless it's a digital listing.
